app/src/main/java/com/example/warehousemanagement/domain/model/Customer.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.warehousemanagement.domain.model
+
+data class Customer(
+    val id: String,
+    val customerName: String,
+    val address: Address,
+
```

Comment:
Remove this redundant line ( and reformat, in mac is `command` + `option`+ `l`. Please research in Win

Reply:
Done